### ISSUE TITLE
when I use ES6 to import Clipboard from 'clipboard', There is an error occurred that ClipBoard is undefined

### ISSUE THREAD

**Issue by onedaywen** [zenorocha/clipboard.js#523]
### Minimal example

> I was running the project use webpack bundle, So I can not afford the JSFiddle link.
### Tech stack
wepback + babel-loader + ES6

at first
```
npm i clipboard --save
```
then
```
import ClipBoard from 'clipboard';
console.log(ClipBoard);
```
### Expected behaviour
It is expected to console.log an contructor function

### Actual behaviour

console.log undefined

### At last
plz let me know why this happen, it is my honor if I can help.

**Comment by jchook** [comment]
I am having this problem too.

Did you fix it?

**Comment by zenorocha** [comment]
This should be fixed with `2.0.1`, can you guys test and let me know?

**Comment by arozd** [comment]
@zenorocha but version 2.0.1 doesn't work for old browsers :( Do you have any solution for lover version of Clipboard?
